### PR TITLE
Update docs for goToSlide

### PR DESCRIPTION
**Pull request by ericlindley-g**
Not sure if you've already put some of this together, @camelburrito , but I made some quick documentation for the goToSlide method.

/to @camelburrito & @bpaduch for review

### CODE REVIEW

File: spec/amp-actions-and-events.md [hunk 1]
```diff
@@ -147,6 +147,11 @@ For example, the following is possible in AMP.
     <td>dismiss (default)</td>
     <td>Hides the referenced user notification element</td>
   </tr>
+  <tr>
+    <td width="30%">amp-carousel</td>
+    <td>goToSlide</td>
+    <td>Advances the carousel to a specified slide index</td>
```
Comment (camelburrito) [inline spec/amp-actions-and-events.md]: type="slides"

File: extensions/amp-carousel/amp-carousel.md [hunk 2]
```diff
@@ -57,6 +57,46 @@ The carousel advances between items if the user swipes, uses arrow keys, or clic
 </amp-carousel>
 ```
 
+### Advancing the carousel to a specific slide
```
Comment (camelburrito) [inline extensions/amp-carousel/amp-carousel.md]: you will need to specify this works for type="slides" only

**Comment by camelburrito** [comment]
LG other than 2 comments

File: spec/amp-actions-and-events.md (hunk 1 above)
Comment (ericlindley-g) [inline spec/amp-actions-and-events.md]: Updated

File: extensions/amp-carousel/amp-carousel.md (hunk 2 above)
Comment (ericlindley-g) [inline extensions/amp-carousel/amp-carousel.md]: Updated — thanks!

**Comment by ericlindley-g** [comment]
@camelburrito Made the changes (good eye :)  and ready to merge. Thanks!

**Pull request merged**